src/components/manager/ManagerExportContent.js

```diff
@@ -2,15 +2,30 @@ import React from "react";
 import { useDispatch } from "react-redux";
 import { useNavigate } from "react-router-dom";
 
+import { exportData, RESOURCE_EVERYTHING } from "bento-auth-js";
+
+import { useResourcePermissionsWrapper } from "@/hooks";
 import { submitExportWorkflowRun } from "@/modules/wes/actions";
 
+import ForbiddenContent from "./ForbiddenContent";
 import RunSetupWizard from "./RunSetupWizard";
 import RunSetupConfirmDisplay from "./RunSetupConfirmDisplay";
 
 const ManagerExportContent = () => {
     const dispatch = useDispatch();
     const navigate = useNavigate();
 
+    const { permissions, hasAttemptedPermissions } = useResourcePermissionsWrapper(RESOURCE_EVERYTHING);
+
+    // TODO: each workflow should have definitions for permissions scopes, so we can instead check if we can run at
+    //  least one workflow.
+
+    if (hasAttemptedPermissions && !permissions.includes(exportData)) {
+        return (
+            <ForbiddenContent message="You do not have permission to view the export wizard." />
+        );
+    }
```

Comment:
Can be simplified with `useHasResourcePermissionWrapper(RESOURCE_EVERYTHING, analyzeData)`.

Reply:
same here